src/routes/createGame.ts

```diff
@@ -29,63 +29,61 @@ router.get(
     }),
   }),
   async (req, res) => {
-    const creatorToken = req.query.creatorToken as string
-    const nickname = req.query.nickname as string
-    const smallBlind =
-      req.query.smallBlind === undefined
-        ? SMALL_BLIND_DEFAULT.toString()
-        : (req.query.smallBlind as string)
-    const startingFunds =
-      req.query.startingFunds === undefined
-        ? STARTING_FUNDS_DEFAULT.toString()
-        : (req.query.startingFunds as string)
+    const creator: FirebasePlayerInfo = {
```

Comment:
Well, now I wonder if the name `FirebasePlayerInfo` is a correct name for this interface, as we now use it in differrent context.

Reply:
That also crossed my mind - maybe just `BasicPlayerInfo`?

Reply:
Definitely better than the previous one.